ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```diff
@@ -9,6 +9,11 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.11
+  hive:
+    cluster: https://api.ci.openshift.org
```

Comment:
`cluster` does not need to be there (ideally, remove it also from the other two items)

Reply:
Done.